Lib/enum.py

```diff
@@ -481,18 +482,149 @@ def __new__(metacls, cls, bases, classdict, *, boundary=None, _simple=False, **k
         if exc is not None:
             raise exc
         #
+        # update classdict with any changes made by __init_subclass__
+        classdict.update(enum_class.__dict__)
+        #
+        # create a default docstring if one has not been provided
+        if enum_class.__doc__ is None:
+            if not member_names:
+                enum_class.__doc__ = classdict['__doc__'] = _dedent("""\
```

Comment:
Maybe it would be more readable to move this string to a module level?

Reply:
I prefer to keep the two together -- makes future updates easier.